analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -76,18 +76,22 @@
     '-fext-.*-literals',
     '-ffixed-r2',
     '-ffp$',
+    '-mfp16-format',
     '-fgcse-lm',
     '-fhoist-adjacent-loads',
     '-findirect-inlining',
     '-finline-limit',
     '-finline-local-initialisers',
     '-fipa-sra',
+    '-fmacro-prefix-map',
     '-fno-aggressive-loop-optimizations',
     '-fno-canonical-system-headers',
     '-fno-delete-null-pointer-checks',
+    '-fno-defer-pop',
     '-fno-extended-identifiers',
     '-fno-jump-table',
     '-fno-keep-static-consts',
+    '-fno-reorder-functions',
```

Comment:
Note: on the Phabricator there is a patch for clang to support this flag but as far as I see it is not merged yet and still under review: https://reviews.llvm.org/D34796.

Reply:
Thanks for your response @csordasmarton. I have removed this flag altogether because it is already covered by the flag in the next line `-f(no-)?reorder-functions`